tests/simulator/py/refinement_boxes.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+#
+# formatted with black
+
+import unittest, os, phare.pharein as ph
+from datetime import datetime, timezone
+from ddt import ddt, data
+from phare import populateDict
+from tests.simulator import test_simulator as tst
+from tests.simulator.py import basicSimulatorArgs, makeBasicModel
+
+out = "phare_outputs/valid/refinement_boxes/"
+diags = {"diag_options": {"format": "phareh5", "options": {"dir": out}}}
+
+
+@ddt
+class SimulatorRefineBoxInputsB(unittest.TestCase):
+    def dup(dic):
+        dic.update(diags.copy())
+        return dic
+
+    valid1D = [
+        dup({"refinement_boxes": {"L0": {"B0": [(10,), (50,)]}}}),
```

Comment:
can we had some more cases? like more than one refinement box? what's the smallest refinement box possible? etc.? with in mind to try to break the code and put in the valid/invalid cases real examples of what's possible or not?

Reply:
regarding breaking the code, I would think we'd want the field validation/initialization in there to see just at what point do things start to get a bit weird